# About me

<div style="display: flex; flex-wrap: wrap; align-items: center;">
  <img src="/assets/zUg6W86__400x400.jpg" alt="Alan Nawzad Amin" align="left" width="287" style="max-width: 50%; height: auto; margin-right: 10px; display: block;">

  <div style="flex: 1; min-width: 200px;">
    <p>I'm Alan Nawzad Amin!
    I build statistical tools for all types of biological sequence data, especially proteins, transcriptomics, and genomics.
    In particular, I build models and tests that 1) scale to massive modern datasets, 2) are provably flexible enough to fit this data, and 3) incorporate biological assumptions.</p>
    <p><strong>Bio:</strong> I am a faculty fellow Courant Institute at NYU, working with the <a href="https://cims.nyu.edu/~andrewgw/">Wilson lab</a>.
    In the summer of 2023, I was a postdoc at <a href="https://www.jura.bio">Jura bio</a>.
    I completed my PhD in the Harvard Systems Biology program supervised by <a href="https://www.deboramarkslab.com/">Debora Marks</a> in 2023.
    I graduated with a BS in Biochemistry and Mathematics from the University of Toronto in 2019.
    <a href="https://github.com/AlanNawzadAmin/alannawzadamin.github.io/blob/main/assets/Alan_Amin_CV.pdf?raw=true">A link to my CV.</a>
    <a href="https://github.com/AlanNawzadAmin/alannawzadamin.github.io/blob/main/assets/Research_statement.pdf?raw=true">A link to a research statement (2023).</a>
    </p>
  </div>
</div>



# Selected works
\* denotes equal contribution

### Building scalable, flexible models of sequence data

Scalable and Flexible Causal Discovery with an Efficient Test for Adjacency.  
&nbsp;&nbsp;&nbsp;&nbsp;**Amin A N**, Wilson A G. *ICML*, 2024 [paper](https://arxiv.org/abs/2406.09177), [code](https://github.com/AlanNawzadAmin/DAT-graph)

Kernels with Guaranteed Flexibility for Reliable Machine Learning on Biological Sequences.  
&nbsp;&nbsp;&nbsp;&nbsp;**Amin A N**, Weinstein E N\*, Marks D S\*. *arXiv*, 2023. **Student Research Award at 2023 New England Statistics Symposium** [paper](https://arxiv.org/abs/2304.03775), [code](https://github.com/AlanNawzadAmin/Kernels-with-guarantees)

A generative nonparametric Bayesian model for whole genomes.  
&nbsp;&nbsp;&nbsp;&nbsp;**Amin A N\***, Weinstein E N\*, Marks D S. *NeurIPS*, 2021. [paper](https://proceedings.neurips.cc/paper/2021/hash/e9dcb63ca828d0e00cd05b445099ed2e-Abstract.html), [code](https://github.com/debbiemarkslab/BEAR)

### Evaluating models of sequence data

Kernel-Based Evaluation of Conditional Biological Sequence Models.  
&nbsp;&nbsp;&nbsp;&nbsp;Glaser P, Paul S, Hummer A M, Deane C M, Marks D S, **Amin A N**. *ICML*, 2024.

A Kernelized Stein Discrepancy for Biological Sequences.  
&nbsp;&nbsp;&nbsp;&nbsp;**Amin A N**, Weinstein E N\*, Marks D S\*. *ICML*, 2023. [paper](https://proceedings.mlr.press/v202/amin23a.html), [code](https://github.com/AlanNawzadAmin/KSD-B/)

Non-identifiability and the blessings of misspecification in models of molecular fitness and phylogeny.  
&nbsp;&nbsp;&nbsp;&nbsp;Weinstein E N\*, **Amin A N\***, Frazer J, Marks D S. *NeurIPS*, 2022. **(Oral)** [paper](https://proceedings.neurips.cc/paper_files/paper/2022/file/247e592848391fe01f153f179c595090-Paper-Conference.pdf)

# Teaching

CSCI-102: Data structures Fall 2024. *Reach out if you're an undergrad at NYU and would like to be a grader this fall!*

CSCI-102: Data structures Spring 2024. [Website](https://github.com/AlanNawzadAmin/CSCI-UA-201-011-Spring-2024)

# Contact
Please feel free to reach out to me at alanamin@nyu.edu!


