
# About me

<p align="left">
  <img src="/assets/zUg6W86__400x400.jpg" align="left" width="300" style="margin-right: 10px;">
  I'm Alan Nawzad Amin!
  I build statistical tools for all types of biological sequence data, especially proteins, transcriptomics, and genomics.
  In particular, I build models and tests that 1) scale to massive modern datasets, 2) are provably flexible enough to fit this data, and 3) incorporate biological assumptions.
  
  **Bio:** I am a faculty fellow Courant Institute at NYU, working with the [Wilson lab](https://cims.nyu.edu/~andrewgw/).
  In the summer of 2023, I was a postdoc at [Jura bio](https://www.jura.bio).
  I completed my PhD in the Harvard Systems Biology program supervised by [Debora Marks](https://www.deboramarkslab.com/) in 2023.
  I graduated with a BS in Biochemistry and Mathematics from the University of Toronto in 2019.
</p>

[A link to my CV.](https://github.com/AlanNawzadAmin/alannawzadamin.github.io/blob/main/assets/Alan_Amin_CV.pdf?raw=true)

[A link to a research statement (2023).](https://github.com/AlanNawzadAmin/alannawzadamin.github.io/blob/main/assets/Research_statement.pdf?raw=true)

# Selected works
\* denotes equal contribution

### Building scalable, flexible models of sequence data

**Amin A N**, Wilson A G. Scalable and Flexible Causal Discovery with an Efficient Test for Adjacency. ICML, 2024 [paper](https://arxiv.org/abs/2406.09177), [code](https://github.com/AlanNawzadAmin/DAT-graph)

**Amin A N**, Weinstein E N\*, Marks D S\*. Kernels with Guaranteed Flexibility for Reliable Machine Learning on Biological Sequences. arXiv, 2023. **MassMutual Student Research Award at 2023 New England Statistics Symposium** [paper](https://arxiv.org/abs/2304.03775), [code](https://github.com/AlanNawzadAmin/Kernels-with-guarantees)

**Amin A N\***, Weinstein E N\*, Marks D S. A generative nonparametric Bayesian model for whole genomes. NeurIPS, 2021. [paper](https://proceedings.neurips.cc/paper/2021/hash/e9dcb63ca828d0e00cd05b445099ed2e-Abstract.html), [code](https://github.com/debbiemarkslab/BEAR)

### Evaluating models of sequence data

Glaser P, Paul S, Hummer A M, Deane C M, Marks D S, **Amin A N**, Kernel-Based Evaluation of Conditional Biological Sequence Models. ICML, 2024.

**Amin A N**, Weinstein E N\*, Marks D S\*. A Kernelized Stein Discrepancy for Biological Sequences. ICML, 2023. [paper](https://proceedings.mlr.press/v202/amin23a.html), [code](https://github.com/AlanNawzadAmin/KSD-B/)

Weinstein E N\*, **Amin A N\***, Frazer J, Marks D S. Non-identifiability and the blessings of misspecification in models of molecular fitness and phylogeny. NeurIPS, 2022. **(Oral)** [paper](https://proceedings.neurips.cc/paper_files/paper/2022/file/247e592848391fe01f153f179c595090-Paper-Conference.pdf)

# Teaching

CSCI-102: Data structures Fall 2024. *Reach out if you're an undergrad at NYU and would like to be a grader this fall!*

CSCI-102: Data structures Spring 2024. [Website](https://github.com/AlanNawzadAmin/CSCI-UA-201-011-Spring-2024)

# Contact
Please feel free to reach out to me at alanamin@nyu.edu!


