# Alan Nawzad Amin

<div style="display: flex; flex-wrap: wrap; align-items: center;">
  <img src="/assets/zUg6W86__400x400.jpg" alt="Alan Nawzad Amin" align="left" width="302" style="max-width: 50%; height: auto; margin-right: 10px; display: block;">

  <div style="flex: 1; min-width: 200px;">
    <p>I'm Alan Nawzad Amin!
    I work on statistical models built on modern, large databases of biological sequences.
    My projects either build models that can leverage these datasets in new ways, or seek to understand why these models work.</p>
    <p><strong>Bio:</strong> I am a faculty fellow at the Courant Institute at New York University, working with the <a href="https://cims.nyu.edu/~andrewgw/">Wilson lab</a>.
    In the summer of 2023, I was a postdoc at <a href="https://www.jura.bio">Jura bio</a>.
    I completed my PhD in the Harvard Systems Biology program supervised by <a href="https://www.deboramarkslab.com/">Debora Marks</a> in 2023.
    I graduated with a BS in Biochemistry and Mathematics from the University of Toronto in 2019.
    <a href="https://github.com/AlanNawzadAmin/alannawzadamin.github.io/blob/main/assets/Alan_Amin_CV.pdf?raw=true">A link to my CV.</a>
    <a href="https://github.com/AlanNawzadAmin/alannawzadamin.github.io/blob/main/assets/Research_statement.pdf?raw=true">A link to a research statement (2023).</a>
    <p><strong>Contact:</strong> Please feel free to reach out to me at alanamin@nyu.edu! On <a href="https://x.com/AlanNawzadAmin">Twitter @AlanNawzadAmin</a>.</p>
    </p>
  </div>
</div>



# Selected works
\* denotes equal contribution

### Building scalable, flexible models of large sequence data

Training Flexible Models of Genetic Variant Effects from Functional Annotations using Accelerated Linear Algebra. [paper](), [code](https://github.com/AlanNawzadAmin/DeepWAS)  
&nbsp;&nbsp;&nbsp;&nbsp;**Amin A N**, Potapczynski A, Wilson A G.  *ICML*, 2025. **oral and 2nd best paper award at AI4NA workshop at ICLR 2024**

Bayesian Optimization of Antibodies Informed by a Generative Model of Evolving Sequences. [paper](https://arxiv.org/abs/2412.07763), [code](https://github.com/AlanNawzadAmin/CloneBO)  
&nbsp;&nbsp;&nbsp;&nbsp;**Amin A N**, Gruver N\*, Kuang Y\*, Li Y\*, Elliott H, McCarter C, Raghu A, Greenside P, Wilson A G.  *ICLR*, 2024. **spotlight at ICLR, spotlight and outstanding poster award at AIDrugX workshop at Neurips 2024** 

Manufacturing-Aware Generative Model Architectures Enable Biological Sequence Design and Synthesis at Petascale. [paper](https://www.biorxiv.org/content/10.1101/2024.09.13.612900v2), [code](https://github.com/debbiemarkslab/variational-synthesis)  
&nbsp;&nbsp;&nbsp;&nbsp;Weinstein E N\*, Gollub M G\*, Slabodkin A\*, Gardner C L, Dobbs K, Cui X-B, **Amin A N**, Church G M, Wood E B. *Preprint*, 2024. 

Scalable and Flexible Causal Discovery with an Efficient Test for Adjacency. [paper](https://arxiv.org/abs/2406.09177), [code](https://github.com/AlanNawzadAmin/DAT-graph)  
&nbsp;&nbsp;&nbsp;&nbsp;**Amin A N**, Wilson A G. *ICML*, 2024 

Kernels with Guaranteed Flexibility for Reliable Machine Learning on Biological Sequences. [paper](https://arxiv.org/abs/2304.03775), [code](https://github.com/AlanNawzadAmin/Kernels-with-guarantees)  
&nbsp;&nbsp;&nbsp;&nbsp;**Amin A N**, Weinstein E N\*, Marks D S\*. *arXiv*, 2023. **Student Research Award at 2023 New England Statistics Symposium** 

A generative nonparametric Bayesian model for whole genomes. [paper](https://proceedings.neurips.cc/paper/2021/hash/e9dcb63ca828d0e00cd05b445099ed2e-Abstract.html), [code](https://github.com/debbiemarkslab/BEAR)  
&nbsp;&nbsp;&nbsp;&nbsp;**Amin A N\***, Weinstein E N\*, Marks D S. *NeurIPS*, 2021.

### Understanding and evaluating models of sequence data

Why Masking Diffusion Works: Condition on the Jump Schedule for Improved Discrete Diffusion. [paper](arxiv.org/abs/2506.08316), [code](https://github.com/AlanNawzadAmin/SCUD)  
&nbsp;&nbsp;&nbsp;&nbsp;**Amin A N**, Gruver N, Wilson A G.  *Preprint*, 2025.

Kernel-Based Evaluation of Conditional Biological Sequence Models. [paper](https://openreview.net/pdf?id=2dlmcTXfcY)  
&nbsp;&nbsp;&nbsp;&nbsp;Glaser P, Paul S, Hummer A M, Deane C M, Marks D S, **Amin A N**. *ICML*, 2024.

A Kernelized Stein Discrepancy for Biological Sequences.  [paper](https://proceedings.mlr.press/v202/amin23a.html), [code](https://github.com/AlanNawzadAmin/KSD-B/)  
&nbsp;&nbsp;&nbsp;&nbsp;**Amin A N**, Weinstein E N\*, Marks D S\*. *ICML*, 2023.

Non-identifiability and the blessings of misspecification in models of molecular fitness and phylogeny. [paper](https://proceedings.neurips.cc/paper_files/paper/2022/file/247e592848391fe01f153f179c595090-Paper-Conference.pdf)  
&nbsp;&nbsp;&nbsp;&nbsp;Weinstein E N\*, **Amin A N\***, Frazer J, Marks D S. *NeurIPS*, 2022. **Oral** 

# Teaching

CSCI-102: Data structures Fall 2024. *Reach out if you're an undergrad at NYU and would like to be a grader this fall!*

CSCI-102: Data structures Spring 2024. [Website](https://github.com/AlanNawzadAmin/CSCI-UA-201-011-Spring-2024)


